src/app/graph-structure.ts

```diff
@@ -237,7 +237,7 @@ function combineUrlOptionsWithCurrentUrl(urlOptions:string[]):string[] {
 export function getUrlsFromServiceURL(version:string):string[] {
     let graphLinks = constructGraphLinksFromFullPath(AppComponent.explorerValues.endpointUrl);
     if (!graphLinks) return [];
-    let urls;
+    let entityTags; // all tags inside entities - property, navigationProperty
```

Comment:
I suggest that you call these properties instead of tags. They are structural and navigation properties.